scripts/superdesk-authoring/authoring.js

```diff
@@ -102,7 +102,7 @@
     }
 
     function stripHtml(item) {
-        var fields = ['headline', 'abstract'];
```

Comment:
The ticket talks about the description_text and description_html fields. Why are we removing abstract field here?